### PR TITLE
Dutchhunspell

### PR DESCRIPTION
**Pull request by numerys**
#### Testing the changes
- I tested the changes in this PR: YES

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): YES

#### Local build testing
No architecture dependency

### CODE REVIEW

File: srcpkgs/hunspell-nl_NL/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+short_desc="Dutch nl_NL dictionary for hunspell"
+maintainer="Goran Vukoman <g@numerys.net>"
+license="Revised BSD License and/or CC BY 3.0"
+homepage="https://www.opentaal.org/"
+distfiles="https://github.com/OpenTaal/opentaal-hunspell/archive/refs/tags/2.20.19.tar.gz"
+checksum=0a86ec393450d7070bd69ee83f69c37ff27dbbc5fe684803375f113d7128bd87
+
+do_install() {
+	vinstall ${wrksrc}/opentaal-hunspell-2.20.19/nl.aff 644 /usr/share/hunspell nl_NL.aff
+	vinstall ${wrksrc}/opentaal-hunspell-2.20.19/nl.dic 644 /usr/share/hunspell nl_NL.dic
```
Comment (classabbyamp) [inline srcpkgs/hunspell-nl_NL/template:15]:
```suggestion
	vinstall nl.aff 644 /usr/share/hunspell nl_NL.aff
	vinstall nl.dic 644 /usr/share/hunspell nl_NL.dic
```

vinstall works from $wrksrc by default

**Comment by abenson** [comment]
Please [squash the commits](https://www.git-tower.com/learn/git/faq/git-squash) down a single commit.  Use the conventions in [CONTRIBUTING.md](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#committing-your-changes) for the commit message (i.e. "New package: hunspell-nl_NL-2.20.19.").

File: srcpkgs/hunspell-nl_NL/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'hunspell-nl_NL'
+pkgname=hunspell-nl_NL
+version=2.20.19
+revision=0
+create_wrksrc=yes
+short_desc="Dutch nl_NL dictionary for hunspell"
+maintainer="Goran Vukoman <g@numerys.net>"
+license="Revised BSD License and/or CC BY 3.0"
+homepage="https://www.opentaal.org/"
+distfiles="https://github.com/OpenTaal/opentaal-hunspell/archive/refs/tags/2.20.19.tar.gz"
```
Comment (classabbyamp) [inline srcpkgs/hunspell-nl_NL/template:10]:
```suggestion
distfiles="https://github.com/OpenTaal/opentaal-hunspell/archive/refs/tags/${version}.tar.gz"
```
makes updating easier

File: srcpkgs/hunspell-nl_NL/template [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'hunspell-nl_NL'
+pkgname=hunspell-nl_NL
+version=2.20.19
+revision=0
```
Comment (classabbyamp) [inline srcpkgs/hunspell-nl_NL/template:4]:
```suggestion
revision=1
```
revision always starts at 1

File: srcpkgs/hunspell-nl_NL/template [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+create_wrksrc=yes
```
Comment (classabbyamp) [inline srcpkgs/hunspell-nl_NL/template:5]:
```suggestion
```
shouldn't be needed here

File: srcpkgs/hunspell-nl_NL/template [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+short_desc="Dutch nl_NL dictionary for hunspell"
+maintainer="Goran Vukoman <g@numerys.net>"
+license="Revised BSD License and/or CC BY 3.0"
```
Comment (classabbyamp) [inline srcpkgs/hunspell-nl_NL/template:8]:
```suggestion
license="BSD-3-Clause, CC-BY-3.0"
```
use the [spdx identifiers](https://spdx.org/licenses/)

File: srcpkgs/hunspell-nl_NL/template [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+short_desc="Dutch nl_NL dictionary for hunspell"
+maintainer="Goran Vukoman <g@numerys.net>"
+license="BSD-3-Clause, CC-BY-3.0"
+homepage="https://www.opentaal.org/"
+distfiles="https://github.com/OpenTaal/opentaal-hunspell/archive/refs/tags/${version}.tar.gz"
+checksum=0a86ec393450d7070bd69ee83f69c37ff27dbbc5fe684803375f113d7128bd87
+
+do_install() {
+	vinstall nl.aff 644 /usr/share/hunspell/nl_NL.aff
+	vinstall nl.dic 644 /usr/share/hunspell/nl_NL.dic
```
Comment (classabbyamp) [inline srcpkgs/hunspell-nl_NL/template:14]:
```suggestion
	vinstall nl.aff 644 /usr/share/hunspell nl_NL.aff
	vinstall nl.dic 644 /usr/share/hunspell nl_NL.dic
```

**Review by classabbyamp — COMMENTED** [review]
just this and squash and it should be good

File: srcpkgs/hunspell-nl_NL/template (hunk 6 above)
Comment (numerys) [reply srcpkgs/hunspell-nl_NL/template]: Ok.

**Pull request merged**